src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <title>ASP.NET Core Logging Extensions</title>
-    <Description>This extension enables additional functionality for ASP.NET Core on Azure WebSites, such as enabling Azure logging.</Description>
+    <title>ASP.NET Core Logging Integration</title>
+    <Description>This site extension enables logging integration for ASP.NET Core applications on Azure App Service.</Description>
```

Comment:
While doing https://github.com/aspnet/AspNetCore/issues/4570 I thought that having `extension[s]` in the name is less clear then just integration.

cc @shirhatti @muratg

Reply:
Sounds good to me.

Reply:
Does this surface anywhere in the portal UI? If so, we'll need a doc bug tracking the change in the title

Reply:
https://github.com/aspnet/Docs/issues/9968